trail/upload/models.py

```diff
@@ -481,14 +482,127 @@ def toDict(self):
         metadataDict.update(wcsDicts)
         return metadataDict
 
-    def save(self):
-        """Insert standardized metadata and wcs data into the database."""
-        self.metadata.save()
 
-        # just in case Wcs was appended before metadata, set them explicitly
-        for wcs in self.wcs:
-            wcs.metadata = self.metadata
-            wcs.save()
+@dataclass
+class StandardizedResult:
+    """A dataclass that associates standardized header metadata with one or
+    more standardized WCS and their thumbnails.
+    """
+    _thumbkeys = ["thumbnails", "thumbs", "thumbnail", "thumb"]
+    header: StandardizedHeader = None
+    thumbnails: Sequence[Thumbnails] = field(default_factory=list)
+
+    @classmethod
+    def __resolve_thumbs(cls, data):
```

Comment:
Not on the front page of the gallery, but perhaps on the "single image" view. I could see what it would take to unify the code and use `StandardizedResult`. 

Overall, however, gallery is really all about displaying the thumbs and doesn't care much about metadata or wcs and this class is all about tying the two together so we could, but we wouldn't get that much use out of it yet.